src/pointcloud_to_laserscan_node.cpp

```diff
@@ -152,6 +156,17 @@ void PointCloudToLaserScanNode::cloudCallback(
   scan_msg->range_min = range_min_;
   scan_msg->range_max = range_max_;
 
+  // build pointcloud2 output
+  sensor_msgs::msg::PointCloud2 pointcloud_output;
+  const int no_data = -1;
+  std::vector<int> v_pointcloud_index;
+
+  // build ray output
+  visualization_msgs::msg::Marker ray_output;
+
+  // build stixel output
+  visualization_msgs::msg::MarkerArray stixel_output;
```

Comment:
I think it's better to move these variables(pointcloud_output, ray_output, stixel_output) just before the part they are used in order to reduce the scope.

Reply:
Fixed at https://github.com/tier4/pointcloud_to_laserscan/pull/1/commits/eeed69727f86ca09ea631416fa4277562aad6154